config/config.go

```diff
@@ -44,14 +44,16 @@ type Fetcher struct {
 }
 
 type Config struct {
-	Benchmark   string                  `config:"config.v1.benchmark"`
-	Type        string                  `config:"config.v1.type"`
-	Deployment  string                  `config:"config.v1.deployment"`
-	CloudConfig CloudConfig             `config:"config.v1"`
-	KubeConfig  string                  `config:"kube_config"`
-	Period      time.Duration           `config:"period"`
-	Processors  processors.PluginConfig `config:"processors"`
-	BundlePath  string                  `config:"bundle_path"`
+	Benchmark             string                  `config:"config.v1.benchmark"`
+	Type                  string                  `config:"config.v1.type"`
+	Deployment            string                  `config:"config.v1.deployment"`
+	CloudConfig           CloudConfig             `config:"config.v1"`
+	KubeConfig            string                  `config:"kube_config"`
+	Period                time.Duration           `config:"period"`
+	Processors            processors.PluginConfig `config:"processors"`
+	BundlePath            string                  `config:"bundle_path"`
+	PackagePolicyId       string                  `config:"package_policy_id"`
+	PackagePolicyRevision int                     `config:"revision"`
```

Comment:
any reason to have it as an int?

Reply:
Yes, revision is mapped as short.

Reply:
is there an ECS for it? is it mapped like that throughout Elastic?

Reply:
could not find a related ECS schema.
I think we just followed cloud defend [mapping](https://github.com/elastic/integrations/blob/720540a54f47795a58acfa72508e995dbf028e3e/packages/cloud_defend/data_stream/file/fields/fields.yml#L8C14-L8C14)